tests/types/types.go

```diff
@@ -237,3 +242,72 @@ func GetBaseDisk() []Disk {
 		},
 	}
 }
+
+// Replace all UUID variables (format $uuid<num>) in configs and partitions with an UUID
+func (test *Test) ReplaceAllUUIDVars() error {
+	var err error
+	UUIDmap := make(map[string]string)
+
+	test.Config, err = replaceUUIDVars(test.Config, UUIDmap)
+	if err != nil {
+		return err
+	}
+	for _, disk := range test.In {
+		err = disk.replaceAllUUIDVarsInPartitions(UUIDmap)
```

Comment:
nit: this can be combined as `if err := funcCall(); err != nil {`

Reply:
Why does := work ^? I have err defined above so I'm not sure why the compiler isnt complaining

Reply:
Updated ^

Reply:
Both `if` & `switch` have this functionality inside their initialization statements.

Reply:
so much more fancy than the binary I use to writing :)